test/integration.test.js

```diff
@@ -598,7 +598,7 @@ test("Log unexpected errors from plugins and exit with 1", async (t) => {
   // Verify the type and message are logged
   t.regex(stderr, /Error: a/);
   // Verify the the stacktrace is logged
-  t.regex(stderr, new RegExp(pluginError));
+  t.regex(stderr, new RegExp(process.platform != "win32" ? pluginError : pluginError.replace(/\\/g, "\\\\")));
```

Comment:
Could we make this a strict comparison?  Also, I think I'd prefer reversing to avoid the negation

Reply:
Sorry, ligature fonts may not be as helpful as I had hoped.  It was supposed to be strict.  I will reverse.  I had used negation to keep the non-windows side more visible.